src/utils/router.js

```diff
@@ -106,6 +106,22 @@ export const paths = {
     byNamespace() {
       return byNamespace({ path: '/tasks' });
     }
+  },
+  kubernetesResources: {
+    all() {
+      return '/:group/:version/:type';
+    },
+    byNamespace() {
+      return byNamespace({ path: '/:group/:version/:type' });
+    }
+  },
+  kubernetesResource: {
```

Comment:
Let's merge these into `kubernetesResources` above, no need for a separate object here. Same pattern as used for existing resources.

```
kubernetesResources: {
  all() ...
  byNamespace() ...
  cluster() ...
  byName() ...
}
```